src/shared/settings.js

```diff
@@ -12,17 +12,22 @@ function assign(dest, src) {
 }
 
 /**
- * Return application configuration information from the host page.
```

Comment:
This was actually wrong - the function isn't always used to read settings from the host page